Fixing shipping address override in nxo flow (#662)
braintree/braintree_android: CHANGELOG.md

* BraintreeCore
  * Allow uppercase characters in default return url scheme

* PayPalNativeCheckout (BETA)
  * Bump native-checkout version to `0.8.8`

>   * Adding in a requirement for `returnUrl` inside of `PayPalNativeRequest` this url is needed to redirect after authentication

Reviewer: General question, is this a different return URL than the one that was removed before.

Author: no its the same one, its a requirement on oath 2.0 to have this in so we had to add it back to native

Reviewer: Dang, that's a bummer to have to add it back in. Do you know if we should expect a similar PR for iOS? We should also add this to the docs. Just curious what the experience will be if a merchant doesn't include this? Is this auth change already live or waiting until we do a release to make updates?

Author: Yup we need to make a small change to the dev docs to add this in! I can work on doing that, is there a process for that? I remember there being a git somewhere. IOS will not need this change, its an android only requirement since their SVC can just close without context. There is actually no change on auth here, its all client side with the connect flow. If a merchant does not include this, they will be stuck after auth and will not be redirected to their application, which is expected as thats the current behavior on nxo

Reviewer: Yeah the [docs repo is here](https://github.braintreeps.com/braintree/docs) - you'd need to open up a PR to it with this change, once it's merged it gets auto released by the docs team weekly.

And thanks for clarifying that it's only needed on Android!

Author: thank you! Will open a PR now

>   * Adding in a requirement for `returnUrl` inside of `PayPalNativeRequest` this url is needed to redirect after authentication

Reviewer: ```suggestion
  * Breaking change
    *`PayPalNativeRequest` requires a `returnUrl` to redirect correctly after authentication
```
Thinking we should maybe call this out as a breaking change

Reviewer 2: +1 on the breaking change callout.
